packages/apps/human-app/frontend/src/components/data-entry/select.tsx

```diff
@@ -30,26 +29,22 @@ export function Select({
     <Controller
       name={name}
       render={({ field, fieldState }) => (
-        <FormControl
-          error={Boolean(fieldState.error)}
```

Comment:
error={Boolean(fieldState.error)} shoud exist in component

Reply:
that should not be removed